### ISSUE TITLE
Why use HingeLoss instead of BPRLoss ?

### ISSUE THREAD

**Issue by KID-22** [GanjinZero/RRHF#15]
This is good job. However, we always use BPRLoss rather than HingeLoss in pairwise learning to rank since the margin of HingeLoss is hard to tune. So I wonder whther you have tried the BPRLoss?

**Comment by GanjinZero** [comment]
No, I haven't tried it. I found set margin=0 in margin loss run very well. I think other losses can achieve similar results (but you should try it).
